app/helpers/form_helper.py

```diff
@@ -31,7 +31,7 @@ def get_form_for_location(schema, block_json, location, answer_store, disable_ma
 
         data = deserialise_composition_answers(answers)
 
-        return generate_household_composition_form(schema, block_json, data), None
+        return generate_household_composition_form(schema, block_json, data), {}
```

Comment:
Can we remove the return of the empty dict here. Also the same on line 65. Only line 46 returns values and should be made part of the form.